core/src/main/java/org/elasticsearch/action/search/SearchRequest.java

```diff
@@ -169,6 +169,10 @@ public ActionRequestValidationException validate() {
             validationException =
                 addValidationError("using [from] is not allowed in a scroll context", validationException);
         }
+        if (requestCache != null && requestCache && scroll() != null) {
```

Comment:
So this will not be backported to 6.x and instead we will print a deprecation log?

Reply:
yes that's the plan